packages/web-styles/.storybook/main.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  "framework": "@storybook/html",
+  "stories": [
+    "../**/*.stories.@(js|mdx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    {
+      name: "@storybook/preset-scss",
+      opetions: {
+        sassLoaderOptions: {
```

Comment:
- [ ] typo: property-name should be "cssLoaderOptions" (not sure, but check https://www.npmjs.com/package/@storybook/preset-scss)

Reply:
There are three loaders used by webpack for loading styles: style-loader, css-loader and sass-loader. Each loader can be passed options so the key is correct in this case.